blocks/search-content-source-block/sources/search-api.js

```diff
@@ -5,7 +5,7 @@ export default {
   resolve(contentOptions) {
     const { query, page, 'arc-site': arcSite } = contentOptions;
     if (query) {
-      return `https://search.arcpublishing.com/search?&q=${query}&page=${page || 1}${arcSite ? `&website_id=${arcSite}` : ''}${SEARCH_KEY ? `&key=${SEARCH_KEY}` : ''}`;
+      return `https://search.arcpublishing.com/search?&q=${encodeURIComponent(decodeURIComponent(query))}&page=${page || 1}${arcSite ? `&website_id=${arcSite}` : ''}${SEARCH_KEY ? `&key=${SEARCH_KEY}` : ''}`;
```

Comment:
not sure I understand why the query is being decoded first if we're trying to encode it?

Reply:
We have to encode the query to account for the accent characters being sent to the search API.. But in some cases the query is already encoded, so to account for that we decode and encode it. To test this you can remove the `decodeURIComponent` part of this and then load the search page - http://localhost/search/vid%C3%A8o/?_website=the-gazette and you'll see it's double encoding the query value and returning no results

Reply:
`But in some cases the query is already encoded` didn't know this bit